content/docs/user-guide/gems/reference/aws/aws-gamelift/gem-setup.md

```diff
@@ -86,13 +86,14 @@ For a client application, your game must implement the following use cases to ma
 - `JoinSession`
 - `LeaveSession`
 
-The AWS GameLift Gem provides both [C++ APIs](cpp-api/) and [Scripting](scripting/) to support developers. You can implement these use cases using either method. 
+The AWS GameLift Gem provides both [C++ APIs](cpp-api/) and [scripting](scripting/) to support developers. You can implement these use cases using either method.  
+
 
 ## 4. Integrate server
 
-To establish communication between your server and Amazon GameLift service, you must notify Amazon GameLift that your server is ready and then respond to GameLift notifications.
+To establish communication between your server and Amazon GameLift service, you must notify Amazon GameLift that your server is ready and then have your server respond to GameLift notifications.
 
-For more details on Server Notifications, read [C++ API for AWS GameLift Gem](cpp-api/)
+For more details on GameLift server notifications, refer to [C++ API for the AWS GameLift Gem](cpp-api/#server-notifications).
 
 ## 5. Set up AWS credentials
 
```

Comment:
In the link text below this (not in your changes), fix the link text ("Configuring AWS Credentials for O3DE") by dropping "for O3DE".

Reply:
I also dropped the "in the AWS Core section." based on your suggestion from the last review. I forgot to make that change in these updates.